frontend/public/style/_overrides.scss

```diff
@@ -375,11 +375,11 @@ h6 {
 }
 
 .pf-c-table tr > * {
-  --pf-c-table-cell--FontSize: $font-size-base;
-  --pf-c-table-cell--PaddingLeft: 12px;
-  --pf-c-table-cell--PaddingRight: 12px;
-  --pf-c-table--m-compact-cell--first-last-child--PaddingLeft: 12px;
-  --pf-c-table--m-compact-cell--first-last-child--PaddingRight: 12px;
+  --pf-c-table--cell--FontSize: 16px;
```

Comment:
this means table cell content be bigger than the $font-size-base or 14px, which is the default for other content.

Reply:
Hi @rhamilto, restored `$font-size-base`.  Updated screenshot as well. -thanks